en/guide/migrating-5.md

```diff
@@ -116,6 +114,14 @@ The `res.sendfile()` function has been replaced by a camel-cased version `res.se
 
 The `app.router` object, which was removed in Express 4, has made a comeback in Express 5. In the new version, this object is a just a reference to the base Express router, unlike in Express 3, where an app had to explicitly load it.
 
+The new version of the router contains **breaking changes** to the way paths are parsed:
```

Comment:
These are great, but we should also include and list [changes from `2.0.0-beta.1`](https://github.com/pillarjs/router/pull/102/files#diff-648afe3d986261d8f2015b2b131b0e4a448d4dc6946cfde1a7a836876cee255eR19). In a nutshell, these are:

* New `?`, `*`, and `+` parameter modifiers (e.g.: `/users/:user+` now matches `/users/1` and `/users/1/2`, but _not_ `/users`)
* Special `*` path segment behavior removed. This means that `*` is no longer a valid path and `/foo/*/bar` now matches a literal `*`.
* Regular expressions can only be used in a matching group (e.g.: `/users/\\d+` is now invalid and should be written as `/users/(\\d+)`.

Reply:
Ok, fantastic, I'll update the documentation.

I guess it might be worth stating that Express 5 uses path-to-regexp 6, and linking to the documentation for that. Similar to this page:
http://expressjs.com/en/guide/routing.html#route-paths

Which leads a bit deeper down the rabbit hole - that page itself should possibly be updated, but is not versioned. Any suggestions on how to proceed?

Reply:
+1 to all.

Reply:
The migration guide has been updated with 14ee27 - @nfantone please have a look and let me know if it's okay.

@dougwilson how should we update the routing guide? If we follow the error handling guide (which is also unversioned) there's a section "Starting in Express v5..." - perhaps we could do that?

I think my preference would be to have separate documentation for version 5 versus version 4, but obviously that's a larger undertaking.